prow/plugins.yaml

```diff
@@ -0,0 +1,25 @@
+plugins:
```

Comment:
plugins.yaml and plugins.yaml.tpl should be kept in sync IMO. The same applies for config.yaml

Reply:
Done